src/backend/db/categories.js

```diff
@@ -1,5 +1,10 @@
 import { v4 as uuid } from "uuid";
-
+import comic from "../../assets/images/comic.png";
+import biography from "../../assets/images/biography.png";
+import fiction from "../../assets/images/fiction.png";
+import novel from "../../assets/images/novel.png";
+import scifi from "../../assets/images/scifi.png";
+import selfhelp from "../../assets/images/selfhelp.png";
```

Comment:
instead of writing multiple imports here you can make a js file and export from there `import {comic,biography,...} from assets/images/index.js` just like this

Reply:
okay, will do this.